compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -3092,6 +3101,9 @@ define_print! {
             ty::ClauseKind::ConstEvaluatable(ct) => {
                 p!("the constant `", print(ct), "` can be evaluated")
             }
+            ty::ClauseKind::HostEffect(predicate) => {
+                p!("the trait `", print(predicate.trait_ref), "` is const")
+            }
```

Comment:
idk where this is used, but this description is a bit unclear. (Could also change the message depending on whether this is `Maybe` or `Const` too)

```suggestion
            ty::ClauseKind::HostEffect(predicate) => {
                p!("the trait impl `", print(predicate.trait_ref), "` is const")
            }
```

Reply:
Honestly I think I should just change this to stop using words, and just render it like `T: ~const Trait`.